spec/gkv_spec.rb

```diff
@@ -114,4 +114,20 @@ def clear_db
       expect(db.all).to eq [{ 'ants' => 10 }, { 'bob' => 'pants' }, { 'cants' => 10 }]
     end
   end
+
+  context "on save" do
+    it 'returns a hash' do
+      db.set('hello', 'world')
+      expect(db.save).to be_an_instance_of String
+    end
+
+    it 'can be loaded given a hash' do
+      db.set('hello', 'world')
+      hash = db.save
+      $ITEMS = {}
```

Comment:
We could call the helper function defined at the top of spec here, but this is a very minor caveat. Still lgtm